### PR TITLE
Amigo Compatibility

### PR DESCRIPTION
**Pull request by odudex**
Fixed json typos, removed py_lcd fill_rectangle check, changed full filled character to position 161

### CODE REVIEW

**Review by jreesun — CHANGES_REQUESTED** [review]
Looking good! Just have that one request about the config structure

File: projects/maixpy_twatch/builtin_py/board.py [hunk 1]
```diff
@@ -13,6 +13,10 @@
         "dir": 96,
         "lcd_type": 0
     },
+    "sensor":{
```
Comment (jreesun) [inline projects/maixpy_twatch/builtin_py/board.py:16]: `sensor` is here twice now

**Review by jreesun — APPROVED** [review]
LGTM! Thanks for making those changes.

After I merge this, can you update the MaixPy submodule in your krux branch/PR?

**Comment by odudex** [comment]
MaixPy updated on my krux fork and PR.

**Pull request merged**